### DISCUSSION TITLE
HTMLhint? HTML linter?

### DISCUSSION THREAD

**Discussion by wilnoakl** [phcode-dev/phoenix#1827] [Q&A] [answered]
Is any code checking tool available with Phcode? - HTMLhint doesn't seem to work.

**Comment by abose** [comment, marked as answer]
Phoenix has an intergrated HTML Linter. https://docs.phcode.dev/docs/Features/Problems%20Panel/html-lint/

You do not need to integrate any HTML Lint plugins additionaly. Are you facing any issues with the integrated HTML Linter @wilnoakl ?

**Comment by wilnoakl** [comment]
I am used to CSS HTML Validator (Windows) and I expected to have to press a button to validate. As I understand now, mistakes are spotted as they appear. Thanks!

**Comment by wilnoakl** [comment]
As to why I had asked for HTMLhint / HTML linter in the first place, that's because I was blindly following ChatGPT which asked me to install these.

**Comment by abose** [comment]
Got it, Lint is usually activated on save and at other points. I have resolved this issue as answered. If you need any more help, just comment here or create a new discussion 🙂
